java/Nikiforov-Oleksandr/src/com/zoolatech/lecture6/tasks/_3/UserService.java

```diff
@@ -1,13 +1,17 @@
 package com.zoolatech.lecture6.tasks._3;
 
 public class UserService {
-    public UserRepository repository;
+    private UserRepository repository;
 
     public UserService(UserRepository repository) {
         this.repository = repository;
     }
 
-    public String findUserEmail(String id) throws UserMissingException {
-        return repository.findUserEmail(id);
+    public String findUserEmail(String id){
+        try{
+            return repository.findUserEmail(id);
+        }catch (UserMissingException e) {
+            return e.getMessage();
```

Comment:
exception message is not a valid result of the method call: if I'm calling this method, I'm expecting to get an email and not "smth went wrong".